pkg/gadgets/biotop/gadget.go

```diff
@@ -0,0 +1,202 @@
+// Copyright 2019-2021 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package biotop
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/kinvolk/inspektor-gadget/pkg/gadgets"
+	biotoptracer "github.com/kinvolk/inspektor-gadget/pkg/gadgets/biotop/tracer"
+	"github.com/kinvolk/inspektor-gadget/pkg/gadgets/biotop/types"
+
+	gadgetv1alpha1 "github.com/kinvolk/inspektor-gadget/pkg/apis/gadget/v1alpha1"
+)
+
+const (
+	maxRowsDefault  = 20
+	intervalDefault = 1
+	sortByDefault   = types.ALL
+)
```

Comment:
I don't think they will change but what do you think about defining these default values on a "neutral" place like `pkg/gadgets/biotop/types/types.go` and then using those values from the CLI and the container side?

That was the strategy we used for [fsslower](https://github.com/kinvolk/inspektor-gadget/blob/ceea77517b9bfe7c3665967faec27046fe2208b7/pkg/gadgets/fsslower/types/types.go#L21-L23).

Reply:
It is planned, I want to define them in `cmd/kubectl-gadget/top/top.go` of #500 once merged.

Reply:
Mmmm but doing so you need to redefine then again in the "container" side and that's what I am suggesting to avoid, or what did I miss?

Reply:
Oh OK! I did not understand your suggestion!